engine/src/flutter/display_list/utils/dl_matrix_clip_tracker.cc

```diff
@@ -282,18 +283,31 @@ bool DisplayListMatrixClipState::rrect_covers_cull(
     return false;
   }
   auto outer = content.GetBounds();
-  auto center = outer.GetCenter();
-  auto radii = content.GetRadii().top_left;
-  auto inner = outer.GetSize() * 0.5 - radii;
-  auto scale = 1.0 / radii;
-  for (auto corner : corners) {
-    if (!outer.Contains(corner)) {
-      return false;
+  if (content.GetStyle() == impeller::RoundRect::Style::kContinuous) {
```

Comment:
I'm glad you spotted this and addressed it. I'll review the math later, but I was worried this would get missed.